lib/tasks/quality.rake

```diff
@@ -16,7 +16,7 @@ namespace :quality do
     raise "arguments must be provided in YYYY MM order" if year < month
 
     month_interval = DiscoveryEngine::Quality::MonthInterval.new(year, month)
-    DiscoveryEngine::Quality::SampleQuerySet.new(month_interval).create_and_import
+    DiscoveryEngine::Quality::SampleQuerySet.new("clickstream", month_interval).create_and_import
```

Comment:
Is the idea that we'll eventually need to instantiate an object three times for each table id? Could `DiscoveryEngine::Quality::SampleQuerySet` handle that rather than the rake task?

Reply:
You could add the commits to this PR so that the change can be understood fully.